docs/cli-spec.md

```diff
@@ -0,0 +1,47 @@
+# Signing CLI specficiation
+
+### Inputs
+
+The parameters to the signing client are as follows:
+
+`sign code AzureKeyVault [options] <files>`
+
+Options:
+
+- `-o | --output` - The output file. If omitted, overwrites input. Must be a directory if multiple files are specified.
+- `-b | --base-directory` - Base directory for files to override the working directory.
+- `-f | --force` - Overwrites a signature if it exists.
+- `-m | --max-concurrency` - Maximum concurrency (default is 4)
+- `-fl | --filelist` - Path to file containing paths of files to sign within an archive
```

Comment:
In the past we had discussion about signing globing/multiple files same time for `nuget trusted-signers`. It's of course convenient to have one but may course [unwanted issue](https://github.com/NuGet/Home/issues/10647#issuecomment-801277353). 
After discussing with @dtivel we [decided not to implement](https://github.com/NuGet/Home/issues/10647#issuecomment-801523491) globing/multi files signing. Does same concern still apply here?

Reply:
Not at this time as we must support all of the existing functionality of the legacy sign client, and this is one of the features.